kolibri/plugins/document_pdf_render/assets/src/views/index.vue

```diff
@@ -39,6 +62,9 @@
       mimicFullscreen() {
         return !this.fullscreenAllowed && this.isFullscreen;
       },
+      targetTime() {
+        return this.totalPages * 30;
```

Comment:
I did think about that - the only reason I didn't initially implement it, is because of the way we aggregate previous progress on a content item with new progress. So, we have no guarantee of this, but we could enforce it within the session.